.gitmodules

```diff
@@ -31,6 +31,7 @@
 [submodule "waterbox/llvm-project"]
 	path = waterbox/llvm-project
 	url = https://github.com/llvm/llvm-project.git
+	branch = llvmorg-16.0.0
```

Comment:
Does this actually do anything? Because the branch name is actually `release/16.x` while the tag is called `llvmorg-16.0.0`.

Reply:
The release/16.x isn't for 16.0.0, it's for 16.x (it'll get updates eventually for 16.0.1 and beyond most likely). A tag is a branch itself (kinda). I'm not sure it really does anything here other than give us documentation that was at in fact at exactly 16.0.0.

Reply:
I believe it dictates what `git pull` does when it's ambiguous.